base/cvd/fec/BUILD.bazel

```diff
@@ -5,13 +5,6 @@ package(
 cc_binary(
     name = "fec",
     srcs = [
-        "ext4_utils/ext4_sb.cpp",
-        "ext4_utils/include/ext4_utils/ext4_extents.h",
-        "ext4_utils/include/ext4_utils/ext4.h",
-        "ext4_utils/include/ext4_utils/ext4_kernel_headers.h",
-        "ext4_utils/include/ext4_utils/ext4_sb.h",
-        "ext4_utils/include/ext4_utils/jbd2.h",
-        "ext4_utils/include/ext4_utils/xattr.h",
```

Comment:
Should these files be deleted now if they're no longer being used?

Reply:
Whoops, done